src/smart-components/common/form-renderer.js

```diff
@@ -12,28 +15,24 @@ const buttonPositioning = {
   }
 };
 
-const FormRenderer = ({ componentMapper, formContainer, ...rest }) => (
+const FormRenderer = ({ formContainer, ...rest }) => (
   <div className={ buttonPositioning[formContainer].buttonClassName }>
     <ReactFormRender
-      formFieldsMapper={ {
-        ...formFieldsMapper,
-        componentMapper,
-        [componentTypes.SELECT]: Pf4SelectWrapper
+      componentMapper={ {
```

Comment:
Can you define the mapper outside the component? No need to create new objects with each render.